### PR TITLE
[WIP] PyISPyBClient to be used to read and store data via py-ispyb

### PR DESCRIPTION
**Pull request by IvarsKarpics**

### CODE REVIEW

File: HardwareObjects/PyISPyBClient.py [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+        :returns: The session id of the created session.
+        :rtype: int
+        """
+        result = None
+        
+        #if self.is_ready()
+        if True:
+            #try:
+            if True:
+                session_dict["beamLineName"] = session_dict["beamlineName"]
```
Comment (rhfogh) [inline HardwareObjects/PyISPyBClient.py:210]: Works fine, but could be a one-liner
 session_dict["beamLineName"] = session_dict.pop("beamlineName")

**Review by rhfogh — APPROVED** [review]
Looks fine, but I am not qualified to judge. Someone from V3 should do that.

**Comment by beteva** [comment]
Thanks Ivars

**Comment by IvarsKarpics** [comment]
It is difficult to fix the pr and I am closing this PR and will open a new one

**Pull request closed without merging**